backend/src/interface/api_command.rs

```diff
@@ -43,7 +44,7 @@ pub trait Command {
 pub type AuthInfo = Option<InnerAuthInfo>;
 
 /// Structure containing all information necessary for authenticating a client.
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
Clone -> Copy, Clone

Reply:
cannot / should not be copy as a String is not copy